spec/zoneddatetime.html

```diff
@@ -1358,7 +1374,7 @@ <h1>
             }.
         1. Let _addedDate_ be ? AddISODate(_dateTime_.[[ISOYear]], _dateTime_.[[ISOMonth]], _dateTime_.[[ISODay]], 0, 0, 0, _days_, _overflow_).
         1. Let _dateTimeResult_ be ? CreateTemporalDateTime(_addedDate_.[[Year]], _addedDate_.[[Month]], _addedDate_.[[Day]], _dateTime_.[[ISOHour]], _dateTime_.[[ISOMinute]], _dateTime_.[[ISOSecond]], _dateTime_.[[ISOMillisecond]], _dateTime_.[[ISOMicrosecond]], _dateTime_.[[ISONanosecond]], _calendar_).
-        1. Let _instantResult_ be ? GetInstantFor(_timeZone_, _dateTimeResult_, *"compatible"*).
+        1. Let _instantResult_ be ? GetInstantFor(_timeZoneRec_, _dateTimeResult_, *"compatible"*).
```

Comment:
I thought it'd be redundant to have an assertion before each call of GetInstantFor, since it directly calls GetPossibleInstantsFor which asserts that it's been looked up. So I think I'd prefer to leave this as is, but these assertions can always be added later as an editorial change if clarity requires it.